tasks/main.yml

```diff
@@ -1,3 +1,9 @@
+- name: Ensure that required variables are defined.
+  assert:
+    that:
+      - aem_service_name is defined
+      - aem_service_port is defined
+
```

Comment:
Maybe we could add `fail_msg` here to give a specific hint when these variables are not defined.

Reply:
Done